test.py

```diff
@@ -33,6 +33,12 @@ def test_geocode(self):
             isinstance(result['locations']['Moscow']['place'], list)
         )
 
+    def test_get_map(self):
+        """test get the map"""
+        result = self.loklak.get_map(17.582729, 79.118320, 512, 512, 12, 'Hello')
+        self.assertTrue(result[:8] == '\211PNG\r\n\032\n'and (result[12:16] == 'IHDR'))
```

Comment:
What about checking if the img file can be opened properly? (after saving ofc) rather than testing if the file is present.